bundle/jsky.app.ot/src/main/resources/resources/conf/UIInfo.xml

```diff
@@ -105,7 +105,7 @@
             type="siteQuality"
             imageKey="component.gif"
             shortDescription="This component describes the conditions under which this observation can be observed."
-            uiClassName="jsky.app.ot.gemini.editor.EdCompSiteQuality"/>
+            uiClassName="jsky.app.ot.gemini.editor.sitequality.EdCompSiteQuality"/>
```

Comment:
I'm pretty sure we do, and I'm kind of scared to try it, but I will 😹 .

This updated automatically during the refactor, but I'm going to commit the other PR comment changes and then give it a go even though I feel fairly certain it will fail. I'd love to be surprised, though!